lxc/storage.go

```diff
@@ -460,14 +460,21 @@ func (c *cmdStorageInfo) Run(cmd *cobra.Command, args []string) error {
 	poolusedby[usedbystring] = map[string][]string{}
 
 	/* Build up the usedby map
-	/1.0/{instances,images,profiles}/storagepoolname
+	/1.0/{instances,images,profiles}/storagepoolname and
+	/1.0/storage-pools/<poolname>/<type>/<volname>
 	remove the /1.0/ and build the map based on the resources name as key
 	and resources details as value */
 	for _, v := range pool.UsedBy {
-		bytype := string(strings.Split(v[5:], "/")[0])
-		bywhat := string(strings.Split(v[5:], "/")[1])
-
 		u, _ := url.Parse(v)
+
```

Comment:
Need to check Parse error, and I suppose, skip any bad URLs, otherwise u will nil and will cause a panic.